### DISCUSSION TITLE
Integration Environment Variables Support

### DISCUSSION THREAD

**Discussion by jasonmccallister** [solspace/craft-express-forms#77] [Feature Requests]
### Description

I'm just verifying, but it appears that the integration settings do not use the environment variables and require hard coding into the YAML file. Is this a feature that is on your 

### Steps to reproduce

1. Add environment variables to project
2. Configure integrations to use the environment variables

### Additional info

- Express Forms edition & version: 1.0.4
- Craft version: 3.2.10
- PHP version: 7.3.8
- Database driver & version: MySQL 5.6
- Other plugins & versions (if applicable):

**Comment by kjmartens** [comment]
This is correct... currently it just uses its own Yaml file instead. It's on our list to consolidate these settings better with the regular Project Config file in the future. :)

**Comment by rellafella** [comment]
Is there any update on the status of this enhancement?

**Comment by kjmartens** [comment]
@rellafella Sorry, not at this time.
